syntax-markdown-flavor/syntax-markdown-flavor-github/src/main/java/org/xwiki/contrib/rendering/markdown/flavor/github/internal/DefaultMarkdownGithubConfiguration.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.contrib.rendering.markdown.flavor.github.internal;
+
+import org.xwiki.component.annotation.Component;
+import org.xwiki.contrib.rendering.markdown.flavor.github.MarkdownGithubConfiguration;
+import org.xwiki.contrib.rendering.markdown.markdown12.internal.DefaultMarkdownConfiguration;
+
+import javax.inject.Singleton;
+
+/**
+ * GitHub Flavored Markdown Configuration.
+ *
+ * @version $Id$
+ * @since 8.7
+ */
+@Component
+@Singleton
+public class DefaultMarkdownGithubConfiguration extends DefaultMarkdownConfiguration
```

Comment:
Here you indeed need to define a new `MarkdownConfiguration`, but it should have its own name to be used only for Github Markdown Flavor. And most likely you'd need to do some changes on the DefaultMarkdownConfiguration to be able to inherits from it and change only the `ParserEmulationProfile` value.